frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts

```diff
@@ -1,33 +1,39 @@
 import { browser, ExpectedConditions as until, $ } from 'protractor';
```

Comment:
Please modify the structure of this file to make it similar to that of `install-flow` view file